packages/runtime/container-runtime/src/summaryBlobProtocol.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * This class represents the object instatiation of the header which is optionaly writen at the
```

Comment:
It would be great for compression info to be visible and easily modifiable by storage, such that storage (if it wants to) could uncompress / recompress these blobs. This is mostly due to a fact that storage will compress whole file / snapshot used on boot, and having double compression is not great (and will slow down client on boot).

Reply:
I wonder whether we could put this code to the library common to containerRuntime and also to server storage.